style.css

```diff
@@ -581,7 +424,7 @@ textarea {
 
 .error {
   position: absolute;
-  left: 10rem;
+  left: 5rem;
```

Comment:
- [optional] Please kindly work a bit more on the placement of the error message, I noticed that it's on top of the button element, this is not a good UX. ℹ️